src/declarativecontainerwidgetextension_p.h

```diff
@@ -28,11 +28,11 @@
 #ifndef DECLARATIVECONTAINERWIDGETEXTENSION_P_H
 #define DECLARATIVECONTAINERWIDGETEXTENSION_P_H
 
-#include "declarativewidgets_export.h"
+//#include "declarativewidgets_export.h"
```

Comment:
Since the import/export macro is not needed anymore you can just remove this line